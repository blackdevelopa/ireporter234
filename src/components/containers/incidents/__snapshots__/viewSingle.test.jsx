import React from 'react';
import { shallow } from 'enzyme';
import ViewCard from '../SingleIncidentView';

const props = {
  data: [
    {
      id: 11,
      createdon: '2019-05-01T00:00:00.000Z',
      createdby: 19,
      type: 'redflag',
      location: 'Umuahia',
      status: 'draft',
      images: 'https://picsum.photos/id/170/200/300',
      videos: null,
      comment: 'This is a redflag that happened in Umuahia',
    },
  ],
  type: 'intervention',
};

describe('<ViewSingle />', () => {
  const wrapper = shallow(<ViewCard {...props} />);
  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
