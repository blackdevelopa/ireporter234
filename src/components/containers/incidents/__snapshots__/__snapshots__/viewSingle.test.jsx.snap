// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewSingle /> should render without crashing 1`] = `
<Card
  centered={true}
  href="/intervention/11"
  key="11"
>
  <Image
    as="img"
    src="https://picsum.photos/id/170/200/300"
    ui={false}
    wrapped={true}
  />
  <CardContent>
    <CardHeader>
      Umuahia
    </CardHeader>
    <CardMeta>
      <span
        className="date"
      >
        2019-05-01
      </span>
    </CardMeta>
    <CardDescription>
      This is a redflag that happened in Umuahia
    </CardDescription>
  </CardContent>
</Card>
`;
